src/fragments/start/getting-started/reactnative/add-api.mdx

```diff
@@ -0,0 +1,83 @@
+## Deploy your Amplify sandbox backend
+
+Return to the sandbox link you kept handy from earlier. It should look something like the following.
+
+```
+https://sandbox.amplifyapp.com/deploy/<UUID>
+```
+
+### Log in or create a new AWS account
+
+If you don’t have an AWS account, you will need to create one first:
+1. Select **Create an AWS account**
+2. Once you have an account, select **Login to deploy to AWS**
+3. When logged in, you will be taken to the Amplify Console
+
+### Create app backend
+On the creation form:
+<!-- // spell-checker: disable-next-line -->
+1. Give your app a name. We went with **amplifiedtodo**
+2. Select your preferred deployment region
+3. Click **Confirm deployment**
+
+    ![Create app backend](/images/lib/getting-started/reactnative/connect-to-cloud-create-app-backend.png)
+
+4. The on screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Open admin UI**.
+
+    ![Open admin UI](/images/lib/getting-started/reactnative/connect-to-cloud-open-admin-ui.png)
+
+### Update local project with deployed environment
+
+1. After deployment of authentication, click on **Deployment successful - click for next steps** at the top of the Admin UI
```

Comment:
I could be mistaken but is there a missing step here on deploying authentication?

Reply:
Authentication does not need to be deployed for RN Datastore. So, I have removed that step. I have updated the doc to remove it's mention. Thanks Manoj .